### PR TITLE
chore: add stats and cert fields handling in populate exed data mgt command

### PR DESCRIPTION
**Pull request by DawoudSheraz**
### Description

- Handle certificate headline and description, stat 1 & 2 title and text in ExEd Data CSV mgt command
- Give preference to an alternate title
- Add additional column for title for comparing the data if needed

### CODE REVIEW

File: course_discovery/apps/course_metadata/management/commands/populate_executive_education_data_csv.py [hunk 1]
```diff
@@ -267,12 +285,20 @@ def get_transformed_data(self, partially_filled_csv_dict, product_dict):
...
             'long_description': f"{product_dict['introduction']}{product_dict['isThisCourseForYou']}",
             'lead_capture_form_url': product_dict['lcfURL'],
-
-            'title': partially_filled_csv_dict.get('title') or product_dict['name'],
+            'certificate_header': product_dict['certificate'].get('headline', ''),
+            'certificate_text': product_dict['certificate'].get('blurb', ''),
+            'stat1': stats['stat1'],
+            'stat1_text': stats['stat1Blurb'],
+            'stat2': stats['stat2'],
+            'stat2_text': stats['stat2Blurb'],
```
Comment (azanbinzahid) [inline course_discovery/apps/course_metadata/management/commands/populate_executive_education_data_csv.py:294]: is this how we are aiming to call api update to disco?

Comment (azanbinzahid) [reply course_discovery/apps/course_metadata/management/commands/populate_executive_education_data_csv.py:294]: shouldn't we transform and send a dict instead?

Comment (DawoudSheraz) [reply course_discovery/apps/course_metadata/management/commands/populate_executive_education_data_csv.py:294]: No, that is for addition in CSV. Loader handling is in https://github.com/openedx/course-discovery/pull/3321

File: course_discovery/apps/course_metadata/management/commands/tests/test_populate_executive_education_data_csv.py [hunk 2]
```diff
@@ -81,6 +82,16 @@ class TestPopulateExecutiveEducationDataCsv(CSVLoaderMixin, TestCase):
...
                 ],
+                "certificate": {
+                    "headline": "About the certificate",
+                    "blurb": "how this makes you special"
+                },
+                "stats": {
+                    "stat1": "90%",
+                    "stat1Blurb": "A vast number of special beings take this course",
+                    "stat2": "100 million",
+                    "stat2Blurb": "VC fund"
```
Comment (azanbinzahid) [inline course_discovery/apps/course_metadata/management/commands/tests/test_populate_executive_education_data_csv.py]: can add html tags here

**Review by azanbinzahid — APPROVED** [review]

File: course_discovery/apps/course_metadata/management/commands/tests/test_populate_executive_education_data_csv.py (hunk 2 above)
Comment (DawoudSheraz) [reply course_discovery/apps/course_metadata/management/commands/tests/test_populate_executive_education_data_csv.py]: sure.

**Pull request merged**